bpf/go_nethttp.c

```diff
@@ -148,6 +156,8 @@ int uprobe_WriteHeader(struct pt_regs *ctx) {
         bpf_ringbuf_discard(trace, 0);
         return 0;
     }
+
+    bpf_printk("url %s", trace->path);
```

Comment:
Should this (and the other one below) be bpf_dbg_printk, or more context added (e.g. mention WriteHeader?)

Reply:
Nice catch, it's a leftover from debugging.